docs/source/reference/pysupported.rst

```diff
@@ -552,6 +552,11 @@ Typed List
   functionality or suffer from unexpectedly bad performance, please report
   this, ideally by opening an issue on the Numba issue tracker.
 
+.. warning::
+  Most interfaces listed in ``numba/cext/listobject.h`` are exported,
+  but they're not supported/stable, and these interfaces or underlying
+  implementations could be changed or removed in future without notice.
+
```

Comment:
I think this note is a little out-of-place in the section of the manual that discusses support for Python features. Given that the same warning / disclaimer is present in `listobject.h`, I'd suggest just removing this change.

```suggestion
```

Reply:
Okay, solved as you requested.